lib/fluent/plugin/out_color_stripper.rb

```diff
@@ -8,8 +8,6 @@ class ColorStripperOutput < Output
     def configure(conf)
       super
 
-      @tag = conf.fetch('tag') { raise ArgumentError, 'tag field is required to direct transformed logs to' }
```

Comment:
> Fluentd checks tag is present or not on boot.

Or not?  And what does it do if not?  I am not comfortable removing this short circuit unless we can be 100% sure the user will get a useful error message without it.